modules/terraform/format.go

```diff
@@ -38,9 +49,27 @@ func FormatArgs(options *Options, args ...string) []string {
 		terraformArgs = append(terraformArgs, FormatTerraformLockAsArgs(options.Lock, options.LockTimeout)...)
 	}
 
+	if planFileSupported {
+		// The plan file arg should be last in the terraformArgs slice. Some commands use it as an input (e.g. show, apply)
+		terraformArgs = append(terraformArgs, FormatTerraformPlanFileAsArg(commandType, options.PlanFilePath)...)
+	}
+
 	return terraformArgs
 }
 
+// FormatTerraformPlanFileAsArg formats the out variable as a command-line arg for Terraform (e.g. of the format
+// -out=/some/path/to/plan.out or /some/path/to/plan.out). Only plan supports passing in the plan file as -out; the
+// other commands expect it as the first positional argument.
```

Comment:
NIT: mention this returns an empty list if `outPath` is an empty string.

Reply:
Done: 7a3339e6ae1b0a031913a09312c420f35bce923e